src/select-panel/index.tsx

```diff
@@ -204,6 +207,16 @@ const SelectPanel = () => {
       )}
 
       <ul className="options">
+        {isCreatable && canCreatableAppear ? (
+          <li
+            onClick={handleOnCreateOption}
+            className="select-item creatable"
+            tabIndex={skipIndex === 1 ? 0 : 1}
+            ref={creationRef}
+          >
+            {`${t("create")} "${searchText}"`}
+          </li>
+        ) : null}
```

Comment:
- instead of keeping this on top I would like to follow what `react-select` does and keep it last instead of first
- this way we can remove a lot of clutter here keeping old layout so null on like 237 will be replaced with same component
![Screenshot 2022-06-03 at 9 47 59 PM](https://user-images.githubusercontent.com/5774849/171905554-6d2d3e64-3b52-4c37-9a79-ce365bb26290.png)